### PR TITLE
Improve the quiz10 and test

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

Close y-martinez/PyZombis#15

## Checklist

- [X] Variables, functions and comments are translated to Spanish
- [X] Functions follow underscore notation
- [X] Spell check done & typos fixed
- [X] All python code is PEP8 compliant
- [X] Test coverage with Playwright implemented; locators are Pyhton code
- [X] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

(prefer Playwright recorded video or animated gif)

### CODE REVIEW

File: _sources/quiz/Quiz10.rst [hunk 1]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+
+
+Ejercicio 3
+-----------
+
+.. activecode:: q10_3
+    :nocodelens:
+
+
+    Desarrolle la función llamada ``suma_con_suerte`` que recibe tres números enteros ``a``, ``b`` y ``c``. La función debe devolver la suma de los tres números con la condición de que si uno de los números es el **13**, éste no cuenta en la suma asi como todos los números que se encuentren a su derecha. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:97]:
```suggestion
    Desarrolle la función llamada ``suma_con_suerte`` que recibe tres números enteros ``a``, ``b`` y ``c``. La función debe devolver la suma de los tres números con la condición de que si uno de los números es el **13**, éste no cuenta en la suma así como todos los números que se encuentren a su derecha. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 2]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 5
+-----------
+
+.. activecode:: q10_5
+    :nocodelens:
+
+    Desarrolle la función, ``contar_saludo`` que recibe como parámetro una cadena ``s`` y devuelve el número de veces que aparece la cadena ``"hola"`` en ``s`` |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:173]:
```suggestion
    Desarrolle la función, ``contar_saludo`` que recibe como parámetro una cadena ``s`` y devuelve el número de veces que aparece la cadena ``"hola"`` en ``s``. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 3]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 6
+-----------
+
+.. activecode:: q10_6
+    :nocodelens:
+
+    Desarrolle la función ``gato_perro`` que recibe como parámetro una cadena ``s`` y devuelve ``True`` si en la cadena ``s`` aparece el mismo número de veces la cadena "gato" y la cadena "perro" o devuelve ``False`` en caso contrario. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:209]: Aquí me parece bueno poner las cadenas con el formato de código, justo como en el ejercicio de arriba donde dice
> la cadena `"hola"`

```suggestion
    Desarrolle la función ``gato_perro`` que recibe como parámetro una cadena ``s`` y devuelve ``True`` si en la cadena ``s`` aparece el mismo número de veces la cadena ``"gato"`` y la cadena ``"perro"`` o devuelve ``False`` en caso contrario. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 4]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 7
+-----------
+
+.. activecode:: q10_7
+    :nocodelens:
+
+    Desarrolle la función ``contar_code`` que recibe como parámetro una cadena ``s`` y devuelve el número de veces que aparece la cadena ``"code"`` en ``s`` pero con la condición de que la letra ``'s'`` puede ser intercambiada por cualquier otra. Así, las cadenas ``"coze"`` y ``"coxe"`` deben ser contada igualmente. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:249]:
```suggestion
    Desarrolle la función ``contar_code`` que recibe como parámetro una cadena ``s`` y devuelve el número de veces que aparece la cadena ``"code"`` en ``s`` pero con la condición de que la letra ``'s'`` puede ser intercambiada por cualquier otra. Así, las cadenas ``"coze"`` y ``"coxe"`` deben ser contadas igualmente. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 5]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 8
+-----------
+
+.. activecode:: q10_8
+    :nocodelens:
+
+    Sabiendo que la función ``lower()`` convierte los carácteres de una cadena en minúsculas. Desarrolle una función llamada ``finales_iguales`` que recibe dos cadenas ``a`` y ``b``` como parámetros. La función devolverá ``True`` si al final de la cadena ``a`` se encuentra la cadena ``b`` o si al final de la cadena ``a`` se encuentra la cadena ``b`` |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:290]:
```suggestion
    Sabiendo que la función ``lower()`` convierte los carácteres de una cadena en minúsculas. Desarrolle una función llamada ``finales_iguales`` que recibe dos cadenas ``a`` y ``b``` como parámetros. La función devolverá ``True`` si al final de la cadena ``a`` se encuentra la cadena ``b`` o si al final de la cadena ``b`` se encuentra la cadena ``a``. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 6]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 10
+------------
+
+.. activecode:: q10_10
+    :nocodelens:
+
+    Desarrolle la función ``suma_con_mas_suerte`` que recibe una lista ``numeros`` de números enteros y devolver la suma de los números de lista con la condición de que si uno de los números es el **13**, éste no cuenta en la suma asi como todos los números que se encuentren a su derecha. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:366]:
```suggestion
    Desarrolle la función ``suma_con_mas_suerte`` que recibe una lista ``numeros`` de números enteros y devuelve la suma de los números de la lista con la condición de que si uno de los números es el **13**, éste no cuenta en la suma asi como todos los números que se encuentren a su derecha. |br| |br|
```

File: _sources/quiz/Quiz10.rst [hunk 7]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 11
+------------
+
+.. activecode:: q10_11
+    :nocodelens:
+
+    Desarrolle la función ``tiene_2`` que recibe como parámetro una lista ``numeros`` de números enteros y devolver ``True`` si en la lista aparecen dos números consecutivos y ``False`` en caso contrario. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz10.rst:410]:
```suggestion
    Desarrolle la función ``tiene_2`` que recibe como parámetro una lista ``numeros`` de números enteros y devuelve ``True`` si en la lista aparecen dos números iguales consecutivos y ``False`` en caso contrario. |br| |br|
```

**Review by LeoCumpli21 — APPROVED** [review]
LGTM! :smiley:

File: _sources/quiz/Quiz10.rst [hunk 8]
```diff
@@ -7,485 +7,530 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 8
+-----------
+
+.. activecode:: q10_8
+    :nocodelens:
+
+    Sabiendo que la función ``lower()`` convierte los carácteres de una cadena en minúsculas. Desarrolle una función llamada ``finales_iguales`` que recibe dos cadenas ``a`` y ``b``` como parámetros. La función devolverá ``True`` si al final de la cadena ``a`` se encuentra la cadena ``b`` o si al final de la cadena ``b`` se encuentra la cadena ``a``. |br| |br|
```
Comment (angelasofiaremolinagutierrez) [inline _sources/quiz/Quiz10.rst:290]:
```suggestion
    Sabiendo que la función ``lower()`` convierte los carácteres de una cadena en minúsculas. Desarrolle una función llamada ``finales_iguales`` que recibe dos cadenas ``a`` y ``b`` como parámetros. La función devolverá ``True`` si al final de la cadena ``a`` se encuentra la cadena ``b`` o si al final de la cadena ``b`` se encuentra la cadena ``a``. |br| |br|
```

**Review by angelasofiaremolinagutierrez — CHANGES_REQUESTED** [review]
This is ready to merge! Just add the minor changes I suggested ;)

File: _sources/quiz/Quiz10.rst [hunk 9]
```diff
@@ -7,485 +7,530 @@ Quiz - 10
...
+    myTests().main()
+
+
+Ejercicio 7
+-----------
+
+.. activecode:: q10_7
+    :nocodelens:
+
+    Desarrolle la función ``contar_code`` que recibe como parámetro una cadena ``s`` y devuelve el número de veces que aparece la cadena ``"code"`` en ``s`` pero con la condición de que la letra ``'s'`` puede ser intercambiada por cualquier otra. Así, las cadenas ``"coze"`` y ``"coxe"`` deben ser contadas igualmente. |br| |br|
```
[the remark this answers is not in the document]
Comment (y-martinez) [reply _sources/quiz/Quiz10.rst:249]: You are right! 👍🏼

File: _sources/quiz/Quiz10.rst [hunk 10]
```diff
@@ -7,485 +7,530 @@ Quiz - 10
...
-
-.. activecode:: count_hi
-   :nocodelens:
-
-   E. count_hi |br|
-   conta o número de vezes que aparece a string 'hi' |br|
-   count_hi('abc hi ho') -> 1 |br|
-   count_hi('ABChi hi') -> 2 |br|
-   count_hi('hihi') -> 2 |br|
+    Desarrolle la función ``cerca_de_10`` que recibe un entero ``n`` y devuelve ``True`` cuando ``n`` sea múltiplo de 10, o esté a dos unidades de serlo, o ``False`` en caso contrario. |br| |br|
```
[the remark this answers is not in the document]
Comment (y-martinez) [reply _sources/quiz/Quiz10.rst:16]: Yes, that is a better explanation.

**Review by angelasofiaremolinagutierrez — APPROVED** [review]

File: _sources/quiz/Quiz10.rst [hunk 11]
```diff
@@ -7,485 +7,528 @@ Quiz - 10
...
+    from unittest.gui import TestCaseGui
+
+
+    class myTests(TestCaseGui):
+        def testOne(self):
+            self.assertEqual(construir_fila(3, 1, 8), True, "Esperado: True")
+            self.assertEqual(construir_fila(3, 1, 9), False, "Esperado: False")
+            self.assertEqual(construir_fila(3, 2, 10), True, "Esperado: True")
+            self.assertEqual(construir_fila(3, 2, 8), True, "Esperado: True")
+            self.assertEqual(construir_fila(3, 2, 9), False, "Esperado: False")
```
Comment (NicolasSandoval) [inline _sources/quiz/Quiz10.rst:518]: Check tests.

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**